src/pybind/rados/rados.pyx

```diff
@@ -894,7 +943,7 @@ Rados object in state %s." % self.state)
         # for now and remove it later
         with nogil:
             ret = rados_connect(self.cluster)
-        if ret != 0:
+        if (ret != 0 and ret != -errno.EISCONN):
```

Comment:
i'd suggest remove the parenthesis around condition.

Reply:
please note, this changes the behavior of this method. shall we consider it a bug in the application which calls `Rados.connect()` with an already-connected rados instance?

Reply:
1. I'll remove the parenthesis
2. I'll check more detail about this method

Reply:
No. I'll remove the change.